main.py

```diff
@@ -1,12 +1,169 @@
+
+                if (cli_id.isnumeric()) == False:
+                    print("Use digits, please.")
+                elif (cli_id.isnumeric()) == True:
+                    cli_id = int(cli_id)
+                    print("Customer account info you selected: ", video_store.get_customer(id=cli_id))
+            else:
+                print("Could not select. Please enter the name or ID of an existing customer. ")
+        elif employee_choice == '4': 
+            print(f"Then, let's update the account information inside this collection: {video_store.selected_customer}. ")
+            name = input("Enter the new name for the customer account: ")
+            phone = input("Enter the customer's new phone number: ")
+            postal_code = input("Enter the customer's new postal code: ")
+            for_update = video_store.update_customer(name=name, phone=phone, postal_code=postal_code)
+            mark_intro()
+            print("Here's the new account information: ", for_update) # Here's the new account information: {'id': 198, 'name': 'Maxine Waters', 
+                                                                        # 'phone': '333-333-0000', 'postal_code': '20022',
+                                                                        # 'registered_at': 'Sat, 29 May 2021 01:56:21 GMT', 'videos_checked_out_count': 0}
+        elif employee_choice == '5':
+            video_store.delete_customer()
+            mark_intro()
+            print("This customer account has been deleted. See below: ")
+            mark_intro()
+            for customer in video_store.list_customers():
+                print(customer)
+        elif employee_choice == '6':
+            mark_intro()
+            for video in video_store.list_videos():
+                print(video)
+        elif employee_choice == '7':
+            print("We'll add a new video to the database, then. ")
+            title = input("Enter the video's title: ")
+            release_date = input("Enter the video's release date (format: YYYY-MM-DD): ")
+            total_inventory = input("Enter the number of copies that'll be added to the stock: ")
+            available_inventory = total_inventory
+
+            for_display = video_store.create_video(title=title, release_date=release_date, total_inventory=total_inventory,\
+                                                    available_inventory=available_inventory)
+            mark_intro()
+            print("Video addition details: ", for_display) # {'id': 238}
+        elif employee_choice == '8':
+            choice_param = input("Want to select a video by title or ID? ")
+            if choice_param == "title":
+                title = input("Enter the title of the video you want to work with: ")
+                print("Video info you selected: ", video_store.get_video(title=title))
+            elif choice_param == "id": # address case issue later
```

Comment:
You can use `lower()` to convert the case of the string:

```suggestion
            elif choice_param.lower() == "id": # address case issue later
```

Also, if you'd made the case match the prompt it would have been less of an issue (you asked for `ID` but only accepted `id`).